solutions/src/main/java/ru/sberbank/school/task02/ServiceFactoryImpl.java

```diff
@@ -0,0 +1,20 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.exception.FxConversionException;
+
+public class ServiceFactoryImpl implements ServiceFactory {
+
+
+
+    @Override
+    public FxConversionService getFxConversionService(ExternalQuotesService externalQuotesService) throws FxConversionException {
+
+        if(externalQuotesService == null)
```

Comment:
WhitespaceAround: &#39;if&#39; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-671672329>*